dbms/src/Storages/Transaction/KVStore.h

```diff
@@ -77,6 +81,30 @@ class RegionPersister;
 struct CheckpointInfo;
 using CheckpointInfoPtr = std::shared_ptr<CheckpointInfo>;
 
+enum class PersistRegionReason
+{
+    Debug,
+    UselessAdminCommand, // Does not include passive CompactLog
+    AdminCommand,
+    Flush, // passive CompactLog
+    ProactiveFlush,
+    ApplySnapshotPrevRegion,
+    ApplySnapshotCurRegion,
+    IngestSst
+};
+
+constexpr const char * PersistRegionReasonMap[magic_enum::enum_count<PersistRegionReason>()] = {
+    "debug",
+    "admin cmd useless",
+    "admin raft cmd",
+    "tryFlushRegionData",
+    "ProactiveFlush",
+    "save previous region before apply",
+    "save current region after apply",
+    "ingestsst"};
```

Comment:
```suggestion
    "ingest sst"};
```

Reply:
IngestSST is a admin command in raftstore, we use it as a whole